concepts/string-vs-str/about.md

```diff
@@ -30,7 +30,7 @@ This derives from the following properties:
 - Rust's standard library authors therefore chose not to implement the `Index` trait for strings: while it takes a constant time to index into a byte array, it might take quite a long time to index into a string. This violates the principle of least surprise.
 
 ```rust
-let example = "I'm not entirely ascii! ❤";
+let example = "I'm not entirely ASCII! ❤";
```

Comment:
I confirmed that the comments in the below code block are still accurate (because capital ASCII characters take up the same number of bytes as lowercase ASCII characters)

<img width="810" alt="Screen Shot 2022-05-27 at 1 07 44 PM" src="https://user-images.githubusercontent.com/3699047/170751969-cff043bb-c9bc-4b18-a553-4dd9d9df649e.png">

Reply:
Good that you checked it, but yes: all ASCII characters take a single byte, whether encoded as ASCII or as UTF-8. Changing capitalization in either of these schemes will not affect the encoded length.